src/libretro.cpp

```diff
@@ -62,7 +62,7 @@ static void dummy_log( enum retro_log_level level, const char* fmt, ... )
 static retro_video_refresh_t video_cb;
 static retro_input_poll_t input_poll_cb;
 static retro_environment_t env_cb;
-retro_log_printf_t log_cb = dummy_log;
+extern retro_log_printf_t log_cb = dummy_log;
```

Comment:
There must be one definition for `log_cb` in the codebase, and this one is it. The one in `coreopt.c` should really be `extern` though.

Reply:
I revert the `extern` in `libretro.cpp`.